settings.py

```diff
@@ -27,15 +27,15 @@
 
 DEFAULT_MIDI_SOUNDFONT = 'FluidR3_GM.sf2'
 
-DEFAULT_SETTINGS = f"""
+DEFAULT_SETTINGS = """
 [confgen]
 # Set this value to 'true' if you want steam-dos to re-create
 # DOSBox configuration on every run.
-force = {str(DEFAULT_CONFGEN_FORCE).lower()}
+force = {0}
```

Comment:
Hmm, I am not keen on using numbered placeholders here...

I suggest using placeholder name here (`{force}`) and then specifying it in format function (`.format(force=str(DEFAULT_CONFGEN_FORCE).lower(), …`)

Same for other placeholders in this string.

Reply:
Good idea. That does look better.